app/services/npq/create_or_update_profile.rb

```diff
@@ -39,5 +39,16 @@ def teacher_profile
     def user
       @user ||= npq_validation_data.user
     end
+
+    def schedule
+      case npq_validation_data.npq_course.identifier
```

Comment:
Do we want this schedule to be static? Could an NPQ course belong to a schedule?

Reply:
Ehhh, I think I would prefer it to be static for time being. Really, what this is is a 'default schedule for a course' - which will change every couple months. And I prefer changing it in one place in code, rather than updating each course in each environment. We might revisit this idea soon enough though - changing npq schedule will need to know which schedules a course can have, so maybe we will store that in a database.